### ISSUE TITLE
Please elaborate on clients

### ISSUE THREAD

**Issue by chk1982** [MicrosoftDocs/OfficeDocs-Exchange#565]
Hello,

Regarding the Client Considerations section:

"For Outlook 2010 and Outlook 2013, you only need to delete the Outlook user profile."

There is no mention of Outlook 2016. Please include documentation or link to further documentation for handling Outlook 2016 in tenant to tenant migration. 

Please provide as much detail as possible regarding how to successfully transition Outlook clients tenant to tenant when the UPN/email address does not change. If this requires old mail profiles be deleted, please be as specific as possible as to why, as this is not best practice. Currently autodiscover is designed to not require this action. Unfortunately a lack of documentation is making these transitions unpredictable for partners and administrators.

"For Outlook 2007 and Outlook 2010, when you are restarting, auto-discover will configure the client and rebuild the .OST file."

This is the only time the article makes mention of restarting. Please outline the order of operations for profile deletion, restart, etc. 

"For the Lync client, you will need to add contacts once migration is complete."

Please provide a basic level of explanation or link to further documentation as to why this is necessary.

Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59062d31-9618-f316-c6b9-1ed30319aa7f
* Version Independent ID: b5d5266d-ad7d-bc44-53a3-59aecf46391e
* Content: [How to migrate mailboxes from one Office 365 tenant to another](https://docs.microsoft.com/en-us/exchange/mailbox-migration/migrate-mailboxes-across-tenants)
* Content Source: [Exchange/ExchangeOnline/mailbox-migration/migrate-mailboxes-across-tenants.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/mailbox-migration/migrate-mailboxes-across-tenants.md)
* Service: **exchange-online**
* Product: **exchange**
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by eerstech** [comment]
Any Response to CHK1982 , it has been two weeks ..what happen to outlook 2016?

**Comment by j0rt3g4** [comment]
HI @eerstech  and @chk1982  I'm working on this, just solving this issue requires updates on other parts of the documentation and @AndreaBarr and Manuela are working on it.

**Comment by j0rt3g4** [comment]
I've worked on this request I've added information:
Pull request #680 

The case where the domain source and target are equal it's a new page request and needs to be managed in another issue.

**Comment by chk1982** [comment]
Hi @j0rt3g4 I'm back to working on another major tenant to tenant and wondering about this. Time to include Outlook 2019 as well. Any update? Thanks!
